cfgov/cal/views.py

```diff
@@ -46,6 +66,36 @@ def clean_filter_calendar(self):
         calendars = CFPBCalendar.objects.filter(title__in=calendar_names)
         return calendars
 
+class CalendarPDFForm(forms.Form):
+    filter_calendar = FilterCheckboxList(
+            choices = [(c.title, c.title) for c in CFPBCalendar.objects.all()],
```

Comment:
This usage of `CFPBCalendar.objects.all()` is problematic here because it means you are trying to make a DB query at module import time, which isn't always possible if the database isn't ready.